openshift/templates/react-frontend/dc.yaml

```diff
@@ -34,6 +34,10 @@ objects:
             deploymentconfig: ${NAME}-${SUFFIX}
         spec:
           volumes:
+            - configMap:
+                defaultMode: 420
```

Comment:
What does the defaultMode number mean?

Reply:
Those are the permission the volume file is created with.  (I had to use the command:
`oc explain dc.spec.template.spec.volumes.configMap.defaultMode`
To get an explanation. 420 means the owner of the file can read, group members can write to it, others can do nothing. No one can execute the files in the volume.
I copied the permissions from the other config volume in the system.